rskj-core/src/main/java/co/rsk/rpc/netty/JsonRpcWeb3ServerHandler.java

```diff
@@ -101,6 +103,11 @@ protected void channelRead0(ChannelHandlerContext ctx, ByteBufHolder request) th
             int errorCode = error.getCode();
             responseContent = buildErrorContent(errorCode, error.getMessage());
             responseCode = errorCode;
+        } catch (RskJsonRpcRequestException e) {
```

Comment:
what is the use case catch here? In most of then the exception is handled into the `JsonRPCBasicServer`

Reply:
That will catch the errors thrown by the `JsonRpcWeb3ServerHandlerInvocationListener` we added above.